app/src/main/java/edu/uc/kovaciad/slicetracker/dto/SliceFile.kt

```diff
@@ -1,6 +1,8 @@
 package edu.uc.kovaciad.slicetracker.dto
 
 import android.text.format.Time
+//as of api 22 Time is depreciated and it is suggested to use GregorianCalendar
```

Comment:
I'm not sure if calendar or GregorianCalander is actually the best option, as you are using this variable to store an amount of time something will take, so there will probably have to be some conversions done

Reply:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.